<?xml version="1.0"?>
<launch>
    <arg name="roomba" default="roomba1" />

    <!-- tf_broadcaster -->
    <node pkg="object_localizer" type="tf_broadcaster" name="$(arg roomba)_tf_broadcaster" output="screen" >
        <!-- remap -->
        <remap from="odom_in" to="$(arg roomba)/roomba/odometry" />
        
        <!-- params -->
        <param name="BASE_LINK_FRAME_ID" value="$(arg roomba)/base_link" />
    </node>

</launch>